nautobot/dcim/management/commands/migrate_location_contacts.py

```diff
@@ -0,0 +1,205 @@
+                self.stdout.write(self.style.WARNING("t") + ": Create a new Team")
+                self.stdout.write(self.style.WARNING("s") + ": Skip this location")
+                while True:
+                    choice = input("Select a choice from the list of items: ")
+                    if choice == "s":
+                        self.stdout.write(f"Skipping location {location.display}")
+                        break
+                    elif choice.lower() == "c":
+                        selected_contact = self.create_new_contact_from_location(location, Contact)
+                        break
+                    elif choice.lower() == "t":
+                        selected_contact = self.create_new_contact_from_location(location, Team)
+                        break
+
+            else:
+                # Found similar contacts or teams, prompt user for action
+                self.stdout.write("")
+                self.stdout.write(self.style.WARNING(f"Found similar contacts/teams for location {location.display}:"))
+                self.stdout.write(f"    current contact name: {location.contact_name!r}")
+                self.stdout.write(f"    current contact phone: {location.contact_phone!r}")
+                self.stdout.write(f"    current contact email: {location.contact_email!r}")
+                self.stdout.write("")
+
+                # Output menu of choices of valid contacts/teams
+                for i, contact in enumerate(similar_contacts_and_teams, start=1):
+                    self.stdout.write(f"{self.style.WARNING(i)}: {contact._meta.model_name.title()}: {contact.name}")
+                    self.print_contact_fields(contact, rjust=len(str(i)) + len(contact._meta.model_name) + 2)
+                    self.stdout.write("")
+                self.stdout.write(self.style.WARNING("c") + ": Create a new Contact")
+                self.stdout.write(self.style.WARNING("t") + ": Create a new Team")
+                self.stdout.write(self.style.WARNING("s") + ": Skip this location")
+
+                # Retrieve desired contact/team from user input
+                while True:
+                    choice = input("Select a choice from the list of items: ")
+                    if choice == "s":
+                        self.stdout.write(f"Skipping location {location.display}")
+                        break
+                    elif choice == "c":
+                        selected_contact = self.create_new_contact_from_location(location, model=Contact)
+                        break
+                    elif choice == "t":
+                        selected_contact = self.create_new_contact_from_location(location, model=Team)
+                        break
+                    elif choice.isdigit() and 0 < int(choice) <= len(similar_contacts_and_teams):
+                        selected_contact = similar_contacts_and_teams[int(choice) - 1]
+                        break
```

Comment:
What happens if the user picks an invalid option?

Reply:
It just keeps printing "Select a choice from the list of items: " until you select a valid one